crates/ibc/src/applications/transfer/amount.rs

```diff
@@ -9,7 +9,7 @@ use crate::bigint::U256;
 #[derive(
     Copy, Clone, Debug, Eq, PartialEq, PartialOrd, Ord, Serialize, Deserialize, Display, From, Into,
 )]
-pub struct Amount(U256);
+pub struct Amount(pub U256);
```

Comment:
`Amount` already implements `Into<U256>` (actually derives it using `derive_more::Into`), so you should be able to convert to `U256` directly using that. I don't think we should make the inner type public, because we want to restrict the arithmetic operations that we allow on `Amount` to `checking_{add,sub}`.

Reply:
Thx your relay, this is help me.